Detectors/MUON/Common/src/dcs-ccdb.cxx

```diff
@@ -120,7 +120,10 @@ void makeCCDBEntryForDCS(const std::string ccdbUrl, uint64_t timestamp)
             << o2::muon::subsysname() << " data points to "
             << CcdbDpConfName() << "\n";
 
-  api.storeAsTFileAny(&dpid2DataDesc, CcdbDpConfName(), md, timestamp);
+  uint64_t endOfValidity = 99999999999999;
```

Comment:
@aphecetche why do you need infinite validity? In this way every next object will overlap will all previous, afaik, this will create overheads. Isn't e.g. 1-2 days validity enough?

Reply:
Hi @shahor02 . 
This is the configuration object, determining which DPs to process. 
Chiara